Source/Core/VideoCommon/RenderBase.h

```diff
@@ -133,6 +134,8 @@ class Renderer
 	// Max height/width
 	virtual int GetMaxTextureSize() = 0;
 
+	static Common::Event s_screenshotCompleted;
```

Comment:
In a busy-loop, yes - but we wouldn't want that. Our preference would be to leave the ```Event``` in and get rid of the ```bool```.